src/simtools/camera/single_photon_electron_spectrum.py

```diff
@@ -92,10 +102,21 @@ def write_single_pe_spectrum(self):
             validate_schema_file=None,
         )
 
-    def _derive_spectrum_norm_spe(self):
+    def _derive_spectrum_norm_spe(
+        self, input_spectrum, afterpulse_spectrum, afterpulse_fitted_spectrum
+    ):
         """
         Derive single photon electron spectrum using sim_telarray tool 'norm_spe'.
 
+        Parameters
+        ----------
+        input_spectrum : str
+            Input file with amplitude spectrum.
```

Comment:
Is this the prompt spectrum? Perhaps better to call it that if so.

Reply:
It doesn't have to. It can be prompt + afterpulse, in case no afterpulse spectrum is given. I've added 

`            Input file with amplitude spectrum (prompt spectrum only if afterpulse spectrum is given).`

Reply:
OK!